actiontext/lib/action_text/engine.rb

```diff
@@ -49,10 +49,6 @@ def to_trix_content_attachment_partial_path
       ActiveSupport.on_load(:action_text_content) do
         self.renderer = ApplicationController.renderer
       end
-
-      ActiveSupport.on_load(:action_controller_base) do
-        before_action { ActionText::Content.renderer = ApplicationController.renderer.new(request.env) }
-      end
```

Comment:
/cc @amatsuda ^

Reply:
🎉

Reply:
@amatsuda I am curious to know how is the performance with this PR now? ref. https://github.com/rails/rails/issues/36963